controllers/feeds/getFeedChat.js

```diff
@@ -253,7 +272,15 @@ const getFeedChatService = async (req, res) => {
         });
       }
       const myReaction = b.reaction;
-      finalize(req, myAnonymousId.user_id, myReaction, newGroup, activity_id, constantActor);
+      finalize(
+        req,
+        myAnonymousId.user_id,
+        myReaction,
+        newGroup,
+        activity_id,
+        constantActor,
+        karmaScores
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `finalize` function is now called with the `karmaScores` parameter. This allows the function to update the karma scores for each user, enhancing the functionality of the application.
```diff
+ finalize(
+   req,
+   myAnonymousId.user_id,
+   myReaction,
+   newGroup,
+   activity_id,
+   constantActor,
+   karmaScores
+ );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->